File: importing_data.R
train_metadata <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_train_metadata.csv.gz")
train_lightcurves <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_train_lightcurves.csv.gz")
test_metadata <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_metadata.csv.gz")

test_lightcurves_01 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_01.csv.gz")
test_lightcurves_02 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_02.csv.gz")
test_lightcurves_03 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_03.csv.gz")
test_lightcurves_04 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_04.csv.gz")
test_lightcurves_05 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_05.csv.gz")
test_lightcurves_06 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_06.csv.gz")
test_lightcurves_07 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_07.csv.gz")
test_lightcurves_08 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_08.csv.gz")
test_lightcurves_09 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_09.csv.gz")
test_lightcurves_10 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_10.csv.gz")
test_lightcurves_11 <- read.csv("/home/fa25225/plasticc/data/plasticc_raw/plasticc_test_lightcurves_11.csv.gz")

test_lightcurves <- rbind(test_lightcurves_01,
                          test_lightcurves_02,
                          test_lightcurves_03,
                          test_lightcurves_04,
                          test_lightcurves_05,
                          test_lightcurves_06,
                          test_lightcurves_07,
                          test_lightcurves_08,
                          test_lightcurves_09,
                          test_lightcurves_10,
                          test_lightcurves_11)

File: importing_data.py
import pandas as pd

#importing objects metadata
train_metadata = pd.read_csv(r'C:\Users\flavio\plasticc\data\plasticc_raw\plasticc_train_metadata.csv.gz')
test_metadata = pd.read_csv(r'C:\Users\flavio\plasticc\data\plasticc_raw\plasticc_test_metadata.csv.gz')

#importing training set lightcurves
train_lightcurves = pd.read_csv(r'C:\Users\flavio\plasticc\data\plasticc_raw\plasticc_train_lightcurves.csv.gz')

# importing test set lightcurves
indices = ["01","02","03","04","05","06","07","08","09","10","11"]
df_list = [pd.DataFrame() for i in range(11)]

for i in range(11):
    df_list[i] = pd.read_csv(r'C:\Users\flavio\plasticc\data\plasticc_raw\plasticc_test_lightcurves_' + indices[i] + '.csv.gz')

test_lightcurves = pd.concat(df_list)